pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.depproj

```diff
@@ -66,7 +66,7 @@
 
     <!-- simple check to make sure we don't accidentally pull an old 
          package that should have been merged to the primary package -->
-    <Error Text="Unexpected System package %(_secondaryFilesByPackagePath.NuGetPackageId)" Condition="$([System.String]::new('%(_secondaryFilesByPackagePath.NuGetPackageId)').Contains('System.'))" />
+    <Error Text="Unexpected System package %(_secondaryFilesByPackagePath.NuGetPackageId)" Condition="$([System.String]::new('%(_secondaryFilesByPackagePath.NuGetPackageId)').Contains('System.')) and ('%(_secondaryFilesByPackagePath.NuGetPackageId)' != 'System.ValueTuple')" />
```

Comment:
This is ugly but I am ok with it since it is temporary and I don't expect us to ever have a *standing* exclusion.